datafusion/sqllogictest/test_files/array.slt

```diff
@@ -1063,7 +1116,7 @@ from arrays_values_without_nulls;
 ## array_element (aliases: array_extract, list_extract, list_element)
 
 # array_element error
-query error DataFusion error: Error during planning: No function matches the given name and argument types 'array_element\(Int64, Int64\)'. You might need to add explicit type casts.\n\tCandidate functions:\n\tarray_element\(array, index\)
```

Comment:
The previous error shows the expected argument, it seems better than signature name itself